doc/basics.md

```diff
@@ -1080,10 +1081,10 @@ auto cars = parser.iterate(cars_json);
 cout << cars.at_pointer("/0/tire_pressure/1") << endl; // Prints 39.9
 ```
 
-A JSON Path is a sequence of segments each starting with the '/' character. Within arrays, an integer
+A JSON Pointer path is a sequence of segments each starting with the '/' character. Within arrays, an integer
```

Comment:
Looking at the example, it seems like "integer index" is zero-based. If that's the case, it would be helpful to document that explicitly.

Reply:
Added a mention to the fact that it is zero-indexed. Thank you!